core/domain/summary_services_test.py

```diff
@@ -196,7 +196,8 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
             'status': 'public',
             'tags': [],
             'thumbnail_bg_color': '#a33f40',
-            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'thumbnail_icon_url': ('%s/assets/images/subjects/Lightbulb.svg'
```

Comment:
You know -- it's a bit silly to repeat this all over the place. Can we have a method in test_utils.py that is something like get_static_asset_url(), and then call it as self.get_static_asset_url('/images/subjects/Lightbulb.svg')?